src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -0,0 +1,62 @@
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { StatusTodos } from '../../types/StatusTodo';
+
+type Props = {
+  visibleTodos: Todo[];
+  status: string,
+  onStatusChanges: (e: string) => void;
+};
+
+export const TodoFilter: React.FC<Props> = ({
+  visibleTodos,
+  status,
+  onStatusChanges,
+}) => {
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {`${visibleTodos.filter(todo => !todo.completed).length} items left`}
```

Comment:
Don't calculate values inside JSX. Move the result of the filtering to the variable.